kornia/augmentation/container/augment.py

```diff
@@ -210,8 +210,8 @@ def _update_transform_matrix_by_module(self, module: Module) -> None:
         if self._transformation_matrix_arg == "skip":
             return
         if isinstance(module, (RigidAffineAugmentationBase2D, RigidAffineAugmentationBase3D, AugmentationSequential)):
-            self._transform_matrices.append(module.transform_matrix)
-            # self._update_transform_matrix(module.transform_matrix)
+            # Passed in pointer, allows lazy transformation matrix computation
+            self._transform_matrices.append(module.transform_matrix)  # type: ignore
```

Comment:
i think this can save from type ignore
```suggestion
            if isinstance(module.transform_matrix, Tensor):
                self._transform_matrices.append(module.transform_matrix)
            else:
                 raise ...
```

Reply:
these cases I don't have this kind of context, when we are going to update this we are going to need to be careful -- apart from that, looking at it now I'm not sure why mypy is mad at this line 😅

Reply:
Since it was None when it firstly got updated, it was about the calling order here. We run into the container first and append its matrix (None), then run into its contents and update the matrix accordingly.